data_specification/constants.py

```diff
@@ -50,7 +50,8 @@
 #: Size of header of data spec pointer table produced by DSE, in bytes.
 APP_PTR_TABLE_HEADER_BYTE_SIZE = 8
 #: Size of data spec pointer table produced by DSE, in bytes.
-APP_PTR_TABLE_BYTE_SIZE = APP_PTR_TABLE_HEADER_BYTE_SIZE + MAX_MEM_REGIONS * 4
+APP_PTR_TABLE_BYTE_SIZE = (
+    APP_PTR_TABLE_HEADER_BYTE_SIZE + MAX_MEM_REGIONS * 4 * 3)
```

Comment:
Magic numbers.
The 4 I imagine is bytes to word which is ok
But why the 3?

Reply:
Similar in Constants.java

Reply:
I see it is correct as it covers  "pointer", "n_words" and "checksum" 
But I should not have had to look!

Reply:
Fixed